app/javascript/packs/heroBannerClose.js

```diff
@@ -1,8 +1,6 @@
-'use strict';
-
-function initializeHeroBannerClose() {
-  let bannerWrapper = document.getElementById('hero-html-wrapper');
-  let closeIcon = document.getElementById('js-hero-banner__x');
+const initializeHeroBannerClose = () => {
+  const bannerWrapper = document.getElementById('hero-html-wrapper');
+  const closeIcon = document.getElementById('js-hero-banner__x');
 
   // Currently js-hero-banner__x button icon ID needs to be written into the abstract html
   // In the future this could be extracted so the implementer doesn't need to worry about it.
```

Comment:
This note has been here for a while - as I understand it campaigns will likely eventually be replaced by billboards, but in the meantime it might be useful to move this to the UI/add a reminder in the HTML variant editor to put in the close button